packages/ember-metal-views/lib/renderer.js

```diff
@@ -63,7 +63,7 @@ function Renderer_renderTree(_view, _parentView, _insertAt) {
     if (!contextualElement && parent && parent._childViewsMorph) {
       contextualElement = parent._childViewsMorph.contextualElement;
     }
-    if (!contextualElement && view._didCreateElementWithoutMorph) {
+    if (!contextualElement) {
```

Comment:
The goal of `view._didCreateElementWithoutMorph` was to ensure only `createElement` did not pass a `contextualElement`. `createElement` cannot be made to accept a context element due to semver constraints, but in theory I would prefer to see it accept one. Rendering a view without an explicit context makes little sense.

My worry about removing the check here is that subtle bugs may creep in where the is no contextual element on a relevant morph. With this change, instead of failing hard we cover for the missing context with a body tag.